### DISCUSSION TITLE
如何自定义直连和代理规则

### DISCUSSION THREAD

**Discussion by tianlongJ** [MetaCubeX/ClashMetaForAndroid#256] [Q&A]
不能自定义规则，在订阅的config.yaml中添加了DOMAIN-SUFFIX规则但是没有命中，走了GeoIP(CN)的规则

**Comment by tianlongJ** [comment]
并且全局模式也没效果，应用还是匹配到GeoIP(CN)走了直连

**Reply by 5lonely** [reply]
我全局代理不能测速

**Comment by Davidasx** [comment]
所以有人有办法吗？电脑clash-verge-rev是可以的呀
